addons/event/models/event.py

```diff
@@ -216,6 +217,8 @@ def write(self, vals):
         res = super(event_event, self).write(vals)
         if vals.get('organizer_id'):
             self.message_subscribe([vals['organizer_id']])
+        if set(vals.keys()).intersection(['name', 'address_id', 'date_end', 'date_begin']):
```

Comment:
can you check with this like : 
 if any(key in ['name', 'address_id', 'date_end', 'date_begin'] for key in vals.keys()):